File: C++/CodePracticeInC++/MoveZero.cpp
#include<iostream>

using namespace std;

int main(){
	int a[]={1,0,3,0,2,0};
	int temp[6];
	int x=0;
	for(int i=0;i<6;i++){
		if(a[i]!=0){
			temp[x]=a[i];
			x++;
			
		}
		
		
	}
	
	for(int i=0;i<6;i++){
		if(a[i]==0){
			temp[x++]=a[i];
		}
	}
	
	for(int i=0;i<x;i++){
		cout<<temp[i];
		
	}
	
}

File: Java work/Code Practice/MoveZero.java
import java.util.*;

public class MoveZero {
    public static void Move(int nums[]) {
        int n = nums.length;
        // int temp[] = new int[n];
        // for (int i = 0; i < n - 1; i++) {
        // if (nums[i] != 0) {
        // temp[i] = nums[i];
        // }
        // }
        // int temp2[] = new int[n];
        // int count = 0;
        // for (int i = 0; i < n; i++) {
        // if (nums[i] == 0) {
        // temp2[i] = nums[i];
        // count++;
        // }
        // }
        // for (int i = n - count; i < n - 1; i++) {
        // temp[i] = temp2[i];
        // }
        int temp[] = new int[n];
        int j = 0;
        for (int i = 0; i < n; i++) {
            if (nums[i] != 0) {
                temp[j] = nums[i];
                j++;
            }
        }
        for (int i = 0; i < n; i++) {
            nums[i] = temp[i];
        }

    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter size of array");
        int n = sc.nextInt();
        System.out.println("Enter element of array");
        int nums[] = new int[n];
        for (int i = 0; i < n; i++) {
            nums[i] = sc.nextInt();

        }

        Move(nums);
        for (int i = 0; i < n; i++) {
            System.out.print(nums[i] + " ");
        }

    }

}
